src/models/AssetAttributes.ts

```diff
@@ -0,0 +1,69 @@
+import { Service, ServiceType } from '../ddo/Service'
+import { EncryptionMethod } from '../services/metadata/MetadataService'
+import { MetaData } from "../ddo/MetaData"
+import AssetPrice from './AssetPrice'
+
+
+export class AssetAttributes {    
+
+    static DEFAULT_ENCRYPTION_METHOD = 'PSK-RSA' as EncryptionMethod
```

Comment:
Do we actually support any other encryption method?

Reply:
yes, ECDSA is supported in `Node`